Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.20 AS builder
+RUN apk update && apk add --no-cache \
+    build-base \
+    curl \
+    git \
+    libpcap-dev \
```

Comment:
For what?

Reply:
> For what?
Ok, Removed unnecessary dependencies

build-base + linux headers = build tools + linux/tcp.h
curl & git = getting latest github release code